src/main/java/bio/terra/app/controller/AdminApiController.java

```diff
@@ -55,7 +55,10 @@ public ResponseEntity<JobModel> registerDrsAliases(List<DrsAliasModel> aliases)
     AuthenticatedUserRequest userReq = getAuthenticatedInfo();
     // Make sure the user is an admin by checking for configure action
```

Comment:
You can remove this comment now :) the new action is self-explanatory!

Reply:
Good catch!